socket_client.c

```diff
@@ -0,0 +1,77 @@
+/* uds_client.c */
+#include <sys/types.h>
+/* Pi-hole: A black hole for Internet advertisements
+*  (c) 2017 Pi-hole, LLC (https://pi-hole.net)
+*  Network-wide ad blocking via your own hardware.
+*
+*  FTL Engine
+*  Unix socket connection test program
+*
+*  This file is copyright under the latest version of the EUPL.
+*  Please see LICENSE file for your rights under this license. */
+
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#define BUF 1024
+#define UDS_FILE "/var/run/pihole/FTL.sock"
```

Comment:
`/var/run` is often a symlink to `/run` which in turn is very often `tmpfs` and volatile. So as long as this survives a reboot then we're okay. Otherwise it will have to be created when the FTL daemon starts. It probably already does that, I'm just reading the `c` code through and leaving this note.

Reply:
Thanks, I wanted to add this to the service script (and maybe also move the log into a separate folder at the same time if that is okay with you @dschaper )